tests/integration/conftest.py

```diff
@@ -29,10 +31,27 @@ def conffile(request):
 
 @pytest.fixture(
     scope="session",
-    params=_ALL_TRAINING_MODULES,
+    params=[
```

Comment:
This is too hardcoded for the schedulefree test case. I was thinking more that the parametrization itself would just be passing whole dicts. The assumption is that the base config already has a complete training module dict, so this would only supply the necessary dict entries to be overriden. So we still include the original 
```python
{
    "_target_": "nequip.train.EMALightningModule",
},
{
    "_target_": "nequip.train.EMAConFIGLightningModule",
},
{
    "_target_": "nequip.train.ScheduleFreeLightningModule",
    "optimizer": {
        "_target_": "schedulefree.AdamWScheduleFree",
        "lr": 0.01,
    }
}

```